projects/project-metadata.json

```diff
@@ -5855,7 +5861,13 @@
           }
         ]
       },
-      "license": null,
+      "license": {
```

Comment:
@Aditi8007 As per the project readme, it is licensed under [oneAPI EULA](https://software.intel.com/content/www/us/en/develop/articles/end-user-license-agreement.html). Please update this information in the metadata